portugues/colabora/comdiferencas/comdiferencas.md

```diff
@@ -1,3 +1,4 @@
 Você encontra os dois commits que são o divisores de água entre o aparecimento do bug que tanto lhe incomoda. 
 Você então:
 
+[Compara e corrige os erros para que tudo funcione.](../corrigedif/corrigedif.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.